File: Sem 1/C/C Program/SimpleIntrest.c
#include<stdio.h>

int main(){
	float p,r,t;

	printf("Enter the principal(p):");
	scanf("%f",&p);

	printf("Enter the rate of intrest(r):");
	scanf("%f",&r);
	
	printf("Enter the timeperiod:");
	scanf("%f",&t);

	printf("SI=%f",((p*r*t)/100));

	return 0;
}

File: Sem 2/OOP Lab/Lab 1 (java)/Prograqm/SimpleIntrest.java
import java.util.*;

public class SimpleIntrest 
	{
	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		System.out.print("enter principal:");
		float p=sc.nextFloat();
		System.out.print("enter time:");
		float t=sc.nextFloat();
		System.out.print("enter rate:");
		float r=sc.nextFloat();

		si(p,r,t);
	}	

	static void si(float a,float b, float c)
	{
		double ans=0;
		ans=(a*b*c)/100;
		System.out.println("SI="+ans);
	}
}
